mosquitto.c

```diff
@@ -1021,39 +1056,75 @@ PHP_MOSQUITTO_API void php_mosquitto_log_callback(struct mosquitto *mosq, void *
 
 PHP_MOSQUITTO_API void php_mosquitto_message_callback(struct mosquitto *mosq, void *client_obj, const struct mosquitto_message *message)
 {
-	mosquitto_client_object *object = (mosquitto_client_object *) client_obj;
-	mosquitto_message_object *message_object;
-	zval *retval_ptr = NULL, *message_zval = NULL;
-	zval **params[1];
+    mosquitto_client_object *object = (mosquitto_client_object *) client_obj;
```

Comment:
Would you mind putting the indentation here back how it was? I use tabs, not spaces, in C code. I can do it if not.